static/bidder-info/pubrise.yaml

```diff
@@ -0,0 +1,21 @@
+endpoint: "https://backend.pubrise.ai/"
+maintainer:
+  email: "prebid@pubrise.ai"
+capabilities:
+  site:
+    mediaTypes:
+    - banner
+    - video
+    - native
+  app:
+    mediaTypes:
+    - banner
+    - video
+    - native
+userSync:
+  redirect:
```

Comment:
User sync is not working. It's returning 404 for the following request. Please re-check.
```
https://sync.pubrise.ai/pbserverIframe?gdpr=&gdpr_consent=&ccpa=&gpp=&gpp_sid=&pbserverUrl=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dpubrise%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D%5BUID%5D
```

Reply:
fixed

Reply:
Working now. thanks!